### PR TITLE
Added the Launcher Interface

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #17 partly.

**Changes**: 
Added the Launcher User Interface consisting of customizable settings providing the users the gateway to all the four program modes (namely FOCUS, RELAX, VJ, SERIAL). Essentially, this PR is aimed at porting the UI of Launcher.java class of neurolab-desktop so that contributers can get an entry point to start porting the whole software in a sequential and an organized manner and the process is expedited therefore.

I have commented the rocket animation part for now because I believe it should be added in the subsequent program modes and not in the Launcher Screen itself. Mentors please correct me if I am wrong and I would be happily incorporating the suggested changes if any.

**Screenshot(s) for the changes**: 
Neurolab-Desktop Launcher Screen:
launcher_neurodesktop

Neurolab-Android Launcher Screen:
launcher_neuroandroid

**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@mariobehling @CloudyPadmal @edenyay @iamareebjamal could you please review the PR?

**Comment by jddeep** [comment]
The animation is something which has to showcased at the forefront right now. @mariobehling can tell you more about this. Right now I can see that you are REMOVING the animation. @suresh-gandhi please take this to another fragment/activity. Otherwise the PR seems fine. Cheers!

File: app/src/main/res/layout/content_main.xml
Comment (jddeep) [inline app/src/main/res/layout/content_main.xml:16]: The animation is something which has to showcased at the forefront right now. @mariobehling can tell you more about this. Right now I can see that you are REMOVING the animation. @suresh-gandhi please take this to another fragment/activity. Otherwise the PR seems fine. Cheers!

**Review by jddeep — CHANGES_REQUESTED** [review]
Please look into the above asked changes.

**Review by RohitJain1103 — DISMISSED** [review]
This seems in the correct direction. Good work @suresh-gandhi

File: app/src/main/res/layout/content_main.xml
Comment (suresh-gandhi) [reply app/src/main/res/layout/content_main.xml:16]: @jddeep Thanks for letting me know. I am waiting for the approval of our mentors @CloudyPadmal @mariobehling @iamareebjamal. If they suggest the same I would surely incorporate your changes.

File: app/src/main/res/values/dimens.xml
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Don't duplicate dimens. Use a standard set

File: app/src/main/res/values/strings.xml
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Wrong use of string resources. Just name this string "settings". Same everywhere else

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
@iamareebjamal this removes a current functionality (a required animation) from the app completely. I suggested to move this to another activity which acts as a Parent Activity. What do you say @iamareebjamal @suresh-gandhi

**Comment by iamareebjamal** [comment]
I'm not familiar with what the animation was doing. Can you elaborate?

**Comment by suresh-gandhi** [comment]
@jddeep I am not sure about it. Don't you think rocket animation should be a part of the program modes rather than the Launcher screen itself? This is the reason why I commented the code in my PR instead of removing it. @iamareebjamal After the device is connected to the app, depending on the mental level(or other parameters) of the user, the rocket will move up and down. 
Anyways, mentors only suggest what should be done.

**Comment by jddeep** [comment]
> @jddeep I am not sure about it. Don't you think rocket animation should be a part of the program modes rather than the Launcher screen itself? This is the reason why I commented the code in my PR instead of removing it. @iamareebjamal After the device is connected to the app, depending on the mental level(or other parameters) of the user, the rocket will move up and down.
> Anyways, mentors only suggest what should be done.

@suresh-gandhi Exactly. That is why i am telling to make this a Parent/Main activity and move the animation to a different fragment (for now) instead of deleting it. Cheers!

**Comment by suresh-gandhi** [comment]
@jddeep Since this is the MainActivity (or the "parent" Activity), I think we should keep the Launcher Interface here. To take care of the rocket animation I am adding the animation in separate activities in all the four modes. I guess its the best thing we can do.

**Comment by jddeep** [comment]
@suresh-gandhi Yes you can proceed with that then.

**Comment by iamareebjamal** [comment]
Rather than creating several activities, create one and pass the mode via intent

**Comment by jddeep** [comment]
> Rather than creating several activities, create one and pass the mode via intent

Yes @iamareebjamal is right. For now we should do that. After each program mode gets their own unique functionality we can change that then.

**Comment by suresh-gandhi** [comment]
@iamareebjamal Good suggestion. Thanks.

File: app/src/main/res/values/strings.xml
Comment (suresh-gandhi) [reply app/src/main/res/values/strings.xml]: @iamareebjamal Shouldn't I use string resources for the Settings title?

**Comment by suresh-gandhi** [comment]
@iamareebjamal @CloudyPadmal Changes done. Please review and merge.

File: app/src/main/java/io/neurolab/MainActivity.java
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/MainActivity.java]: Create a single function with 2 params - R.string.relax_toast and RELAX_PROGRAM_MODE

Reuse for 4 listeners

File: app/src/main/java/io/neurolab/ProgramModeActivity.java
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/ProgramModeActivity.java]: Move the move constants to this Activity

**Comment by suresh-gandhi** [comment]
@iamareebjamal did the changes. Please review and merge. Let me know if anything else needs to be changed.

**Review by iamareebjamal — APPROVED** [review]

**Comment by iamareebjamal** [comment]
Create an issue to set target source compatibility to JDK 8 and use lamdas

**Pull request merged**
